app/bundles/FormBundle/Translations/en_US/messages.ini

```
@@ -200,10 +200,14 @@ mautic.form.table.most.submitted="Most submitted forms"
 mautic.form.table.top.referrers="Top referrers"
 mautic.form.type.header="What type of form do you want to create?"
 mautic.form.type.campaign="Campaign Forms"
-mautic.form.type.campaign.description="Campaign forms can be selected to be the source of contacts when building a campaign. Some submit actions may not be available."
+mautic.form.type.one.differences="<li>Used within campaigns for contact sourcing</li>
+  <li>Limited submit actions</li>
```

```suggestion
  <li>Limited form actions</li>
```

I think we should be consistent across the two options in referring to this as 'form actions' in both places, otherwise folks might be confused between 'submit actions' and 'form actions'? I think the latter is better because it explains that it's actions which relate to the form but we could use 'form submit actions' instead. WDYT?

---

Hm, really nice observation